subsys/bluetooth/mesh/cfg.c

```diff
@@ -100,9 +100,11 @@ int bt_mesh_gatt_proxy_set(enum bt_mesh_feat_state gatt_proxy)
 		return err;
 	}
 
-	if (gatt_proxy == BT_MESH_FEATURE_DISABLED &&
-	    !bt_mesh_subnet_find(node_id_is_running, NULL)) {
-		bt_mesh_adv_gatt_update();
+	if ((IS_ENABLED(CONFIG_BT_EXT_ADV) &&
+	    (gatt_proxy == BT_MESH_FEATURE_ENABLED)) ||
+	    (gatt_proxy == BT_MESH_FEATURE_DISABLED &&
+	     !bt_mesh_subnet_find(node_id_is_running, NULL))) {
+		bt_mesh_adv_update();
```

Comment:
```suggestion
		bt_mesh_adv_gatt_update();
```

Reply:
Seems you use some deprecated names. It is even not built.

Reply:
Yes, i saw that there was some toublr there. Will take a look at it

Reply:
Yes, just sloppyness on my side